authenticator-archetype/src/main/resources/archetype-resources/README.rst

```diff
@@ -6,7 +6,7 @@ A custom authenticator plugin for the Curity Identity Server.
 Building the Plugin
 ~~~~~~~~~~~~~~~~~~~
 
-Ensure that you have obtain credentials for the Curity Nexus server, so that the SDK can be downloaded during compilation. If you do not have a credential, open a ticket in the `Curity Developer Portal <https://developer.curity.io/>`_. Once you have the credential, ensure that it is `configured in your Maven settings <https://developer.curity.io/docs/latest/developer-guide/plugins/index.html#access-to-the-curity-release-repository>`_. Then, build the plugin by issue the command ``mvn package``. This will produce a JAR file in the ``target`` directory, which can be installed.
+Ensure that the Curity Nexus server is `configured in your Maven settings <https://developer.curity.io/docs/latest/developer-guide/plugins/index.html#access-to-the-curity-release-repository>`_. Then, build the plugin by issue the command ``mvn package``. This will produce a JAR file in the ``target`` directory, which can be installed.
```

Comment:
great!

Reply:
Did the other READMEs need updating as well?

Reply:
No, this was the only one which was talking about the credentials to the Curity Nexus server.